### PR TITLE
VIH-11018 Change json serializer from Newtonsoft.Json to System.Text.Json

### PR DESCRIPTION
**Pull request by marcogagliardi**
### Jira link
[VIH-11018](https://tools.hmcts.net/jira/browse/VIH-11018)
### Change description

use System.Text.Json apis

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: VideoWeb/VideoWeb.Common/VideoWeb.Common.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="BookingsApi.Client" Version="2.3.8" />
+    <PackageReference Include="BookingsApi.Client" Version="2.3.10-pr-0893-0024" />
```
Comment (oliver-scott) [inline VideoWeb/VideoWeb.Common/VideoWeb.Common.csproj:7]: just a reminder to update before merging

File: charts/vh-video-web/values.dev.template.yaml [hunk 2]
```diff
@@ -10,4 +10,6 @@ java:
         EJUDAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
         EJUDAD__REDIRECTURI: https://${SERVICE_FQDN}/home
         DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
-        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
\ No newline at end of file
+        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
+        VHSERVICES__VIDEOAPIURL: https://vh-video-api-pr-674.dev.platform.hmcts.net/
```
Comment (oliver-scott) [inline charts/vh-video-web/values.dev.template.yaml:14]: just a reminder to revert before merging

**Review by oliver-scott — APPROVED** [review]

File: charts/vh-video-web/values.dev.template.yaml (hunk 2 above)
Comment (marcogagliardi) [reply charts/vh-video-web/values.dev.template.yaml:14]: done

File: VideoWeb/VideoWeb.Common/VideoWeb.Common.csproj (hunk 1 above)
Comment (marcogagliardi) [reply VideoWeb/VideoWeb.Common/VideoWeb.Common.csproj:7]: done

**Review by will-craig — APPROVED** [review]

**Pull request merged**